csrc/preseg_passes/pre_segmenter.cpp

```diff
@@ -30,6 +31,7 @@ namespace nvfuser::preseg_passes {
   OptimizationPass<MarkAliasesPreparePass>::runPass(fusion);
   OptimizationPass<ExactMappedExtentSubstitutionPass>::runPass(fusion);
   OptimizationPass<AllocationDomainPass>::runPass(fusion);
+  OptimizationPass<RemoveBcastSqueeze>::runPass(fusion);
```

Comment:
I agree. I was thinking put this pass as the first pass, then I thought we may need a discussion on how to order these passes. It seems that the current order is based on the time they are being added.

Reply:
Yeah, most passes are independent, so appending to the end makes sense as a default. In this case the new pass probably won't affect others but I don't think it would hurt to put it further up like in first position as you suggest.